_markets/de/buy-resell/products/_archive.mdx

```diff
@@ -1,11 +1,22 @@
-### POS Archiv
+We offer two solutions for archiving data, see their Product Features below and [find more product details here](https://fiskaltrust.de/produkte/).
+MeinFiskal is described with its [technical details here](https://docs.fiskaltrust.cloud/de/docs/posdealers/buy-resell/products/3rd-party/datev-meinfiskal).
 
-The POS Archive is a **cloud service** which provides a searchable, secure, and **audit-proof** database for Middleware receipts. Activating it for a CashBox will also enable its Middleware instance to regularly push its new receipts into the archive, where they'll be securely persisted in a revision-safe storage. Due to german regulations, the data is kept for **at least ten years**.
+| Product Features | DATEV MeinFiskal    | PosArchiv           |
+|:----------------------|:--------------------|:----------------------|
+|Included in the Carefree Bundle |Yes  |Yes
+|Available as single product |No (in progress)  |Yes
+|Setup for multiple PosOperators |No |Yes  |
+|Place of backup |Germany  |Germany
+|Duration of backup |ten years  |ten years
+|Backup content |DFKA-Files (Daily-Closing), Receipts sent by fiskaltrust.Middleware and fiskaly relevant documentation   |Receipts sent by fiskaltrust.Middleware
+|Necessary requirements |Master Data complete and validated. CashBox active and running. No Account existing on MeinFiskal with similar E-mail-Adress, Daily-Closing receipt required  |Entitlement needed, Daily-Closing receipt required
+|Third party involved |Integration of DATEV MeinFiskal to the fiskaltrust.Portal  |No
+|Searchable backup |Yes  |Yes
+|GoBD - Conformity: |Yes  |Yes
+|Export options: full Export |Export of all data sent to the queue of a CashBox in ~.xml or ~.csv format  |Export of all data sent to the queue of a CashBox in ~.xml or ~.csv format
+|Export options: log data of TSE |Export of the log messages generated by the TSE in ~.tar format |Export of the log messages generated by the TSE in ~.tar format
```

Comment:
Same as comment above (for all lines below, no need to add multiple rows to the table)